CHANGELOG.md

```diff
@@ -152,6 +152,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Improved error messages in `replace_sampler` when the `DataLoader` attributes are not included in the signature or the signature is missing optional arguments ([#8519](https://github.com/PyTorchLightning/pytorch-lightning/pull/8519))
 - Moved `DeviceDtypeModuleMixin` and `HyperparametersMixin` mixin to `core` ([#8396](https://github.com/PyTorchLightning/pytorch-lightning/pull/8396))
 - Return the `default_root_dir` as the `log_dir` when the logger is a `LoggerCollection` ([#8187](https://github.com/PyTorchLightning/pytorch-lightning/pull/8187))
+- Remove restriction that loggers can only log on rank 0 ([#8608](https://github.com/PyTorchLightning/pytorch-lightning/pull/8608))
```

Comment:
this should go into a new section of the changelog for v1.5

@carmocca @awaelchli this would be a breaking change in case users implemented a custom logger and didn't explicitly check for rank zero only. i don't see a way to get around this though

Reply:
true, so this should definitely be in the changed section and I would also mention or make clear that the behavior for existing built-in loggers is unchanged. they still log only rank 0.